starboard/linux/x64x11/clang/3.9/cobalt/configuration.py

```diff
@@ -29,6 +29,13 @@ def GetTestFilters(self):
 
   # A map of failing or crashing tests per target.
   __FILTERED_TESTS = {  # pylint: disable=invalid-name
+      'blackbox': [
+        'cancel_sync_loads_when_suspended',
```

Comment:
I assume we should have a TODO to get the rest of these working? Also, we can probably have the same filters for linux-x64x11 modular bb tests and they would start working?

Reply:
Added a tracking bug.

Reply:
As part of this I want to move these filters to a shared location—I'm trying this out in https://github.com/youtube/cobalt/pull/1398 right now, if that works we can move this

Reply:
Don't we run the test on Linux Modular today? 
I think we should disable only for clang 3.9 modular if we can.

Reply:
We don't run black box tests at all for modular linux today, see the linked PR
It looks to me like with these filters in place, modular linux-x64x11 fails just on `test_wasm_basic`, does that pass for this platform or is it not being run?

Reply:
The `test_wasm_basic` passes on clang 3.9 modular

https://github.com/youtube/cobalt/actions/runs/6396825020/job/17363792926

========================================


[1654:10-03 18:30.730...:INFO:black_box_tests.py(143)] Running WasmBasicTest
[1654:10-03 18:30.731...:WARNING:wasm_basic_test.py(42)] Blackbox tests disabled for platform:linux-x64x11-clang-3-9
[1654:10-03 18:30.731...:INFO:black_box_tests.py(148)] Done WasmBasicTest
[1654:10-03 18:30.731...:INFO:black_box_tests.py(142)] 


test_wasm_basic (cobalt.black_box_tests.tests.wasm_basic_test.WasmBasicTest) ... o

Reply:
Got it, let's add all these filters to the shared linux filters and I can add the filter for the wasm test on top of that just for the linux-x64x11 builds, hopefully we can look into these failures soon

Reply:
Ok, moved them in the shared linux modular filter.